packages-next/auth/src/index.ts

```diff
@@ -55,6 +56,11 @@ export function createAuth<GeneratedListTypes extends BaseGeneratedListTypes>(
       }
       return;
     }
+    // In this case, the person is likely logged in but does not have access to the page
```

Comment:
Unless I'm missing something, this logic is unnecessary..?

Reply:
Without this logic you get an infinite redirect loop.